styleguide/components/21-atoms/link/_link.scss

```diff
@@ -6,37 +6,62 @@ a {
   text-decoration: underline;
   cursor: pointer;
 
+  &:hover {
+    @include theme('background-color', 'color-primary--lighten-4', 'link-background-hover');
+    @include theme('color', 'color-primary--darken-2', 'link-color-hover');
+  }
+
   &:focus {
     outline: thin dotted;
   }
 
-  //
-  // @TODO: Add hover state (waiting for design specifications.)
-  //
-  // &:hover {}
+  // Different button link types. These are links styled as a button.
+  &.button-link {
+    @include button;
 
-  // @TODO: Add external-link variant (waiting for design specifications.)
-  // &.external-link {
-  //   &::before {
-  //     @extend %icon-base-css;
-  //     @extend %icon-newtab;
+    display: inline-block;
+    line-height: inherit;
 
-  //     display: inline-block;
-  //     margin-right: .25rem;
-  //     text-decoration: none;
-  //   }
-  // }
+    i {
+      line-height: .7rem;
+    }
+
+    &-primary {
+      @extend %button-primary;
+    }
 
-  &.read-more-link {
-    @include icon('arrow-right', 'after');
+    &-secondary {
+      @extend %button-secondary;
+    }
+
+    &-dark {
+      @extend %button-dark;
+    }
+  }
 
-    line-height: 1rem;
+  // Standalone links (could be used for external links.)
+  &.stand-alone-link {
     text-decoration: none;
 
-    &::after {
-      position: relative;
-      top: .15rem;
-      margin-left: .5rem;
+    i {
+      line-height: normal;
     }
   }
+
+  // Link sizes.
+  &.extra-large-link {
```

Comment:
Are these the same font-sizes as for buttons?
Can we not create 'lager', 'medium', 'small'... classes and reuse them for consistency?

Reply:
Fixed it with mixins. I don't like a lot of classes for styling in the markup...

Reply:
You prefer bloating your compiled css?

Reply:
It belongs in css. It's styling, it should not be in the markup, that's for sure. 
So yeah I prefer reusable mixins for this.

Reply:
So.. 'extra-large-link' is not styling?
a.extra-large-link is the same as a.extra-large.
'link' is redundant.
Just like we removed it on a.button-link.

Reply:
FIne, we can remove the -link part. But it should stay a mixin...